package.json

```diff
@@ -27,7 +27,9 @@
     "build": "node bin/build",
     "coverage": "istanbul -- cover ./node_modules/.bin/nodeunit tests/unit",
     "data": "node scripts/generate-identifier-data",
-    "pretest": "jshint src && jscs src",
+    "jscs": "jscs src",
+    "jshint": "node bin/jshint src",
```

Comment:
Again you clearly miss the point. It's at least stupid to have jshint as a
dependency for its own code. Instead, we use it directly.
On Jun 20, 2015 7:42 PM, "XhmikosR" <xhmikosr@gmail.com> wrote:

> Please check the whole branch and my other PRs. You've missed the point.
> On Jun 20, 2015 7:41 PM, "Luke Page" <notifications@github.com> wrote:
>
>> In package.json
>> <https://github.com/jshint/jshint/pull/2318#discussion_r32885908>:
>>
>> > @@ -27,7 +27,9 @@
>> >      "build": "node bin/build",
>> >      "coverage": "istanbul -- cover ./node_modules/.bin/nodeunit tests/unit",
>> >      "data": "node scripts/generate-identifier-data",
>> > -    "pretest": "jshint src && jscs src",
>> > +    "jscs": "jscs src",
>> > +    "jshint": "node bin/jshint src",
>>
>> I'm not sure about this.. when developing you use jshint as a tool to
>> check its valid javascript. if it is *not* valid, then this will run
>> your invalid code, error, and you won't get the benefit of jshint telling
>> you where you typed something wrong...
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jshint/jshint/pull/2318/files#r32885908>.
>>
>